### ISSUE TITLE
Cannot build `catboostmodel` target

### ISSUE THREAD

**Issue by mkornaukhov03** [catboost/catboost#2450]
Problem:
I cannot build `catboostmodel` target with CMake.
OS: Ubuntu 18.04
CatBoost version: 119ddb6a33edda7eb8b
CMake version: 3.27.0
conan version: 1.59.0
python3 version: 3.8.17
arch: x86_64
pip version: 23.1.2
Steps:
1) clone repo
2) create directory `catboost/build_cb`
3) `catboost/:$ python3 build/build_native.py --build-root-dir=build_cb --targets catboostmodel`
Error:
```
CMake Error at contrib/libs/ibdrv/CMakeLists.linux-x86_64.txt:19 (target_sources):
  Cannot find source file:

    /catboost/contrib/libs/ibdrv/impl.cpp

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .mpp .m .M .mm .ixx .cppm
  .ccm .cxxm .c++m .h .hh .h++ .hm .hpp .hxx .in .txx .f .F .for .f77 .f90
  .f95 .f03 .hip .ispc
Call Stack (most recent call first):
  contrib/libs/ibdrv/CMakeLists.txt:20 (include)


CMake Error at contrib/libs/ibdrv/CMakeLists.linux-x86_64.txt:10 (add_library):
  No SOURCES given to target: contrib-libs-ibdrv
Call Stack (most recent call first):
  contrib/libs/ibdrv/CMakeLists.txt:20 (include)

```

**Comment by mkornaukhov03** [comment]
Possible temporary fix: checkout on `v1.2` tag, build is successful.

**Comment by andrey-khropov** [comment]
Fixed in https://github.com/catboost/catboost/commit/f68bd8c0af356cca521c19f85b8761cf6541456a
